ui/imports/shared/status/StatusGifPopup.qml

```diff
@@ -73,125 +74,144 @@ Popup {
     }
 
     onClosed: {
-        popup.currentCategory = StatusGifPopup.Category.Trending
-        popup.previousCategory = StatusGifPopup.Category.Trending
+        root.currentCategory = StatusGifPopup.Category.Trending
+        root.previousCategory = StatusGifPopup.Category.Trending
 
         if (confirmationPopup.opened) {
             confirmationPopup.close()
         }
     }
 
-    contentItem: ColumnLayout {
+    contentItem: Item {
         anchors.fill: parent
-        spacing: 0
-
-        Item {
-            property int headerMargin: 8
-
-            id: gifHeader
-            Layout.fillWidth: true
-            height: searchBox.height + gifHeader.headerMargin
-
-            SearchBox {
-                id: searchBox
-                placeholderText: qsTr("Search Tenor")
-                enabled: RootStore.isTenorWarningAccepted
-                anchors.right: parent.right
-                anchors.rightMargin: gifHeader.headerMargin
-                anchors.top: parent.top
-                anchors.topMargin: gifHeader.headerMargin
-                anchors.left: parent.left
-                anchors.leftMargin: gifHeader.headerMargin
-                Keys.onReleased: {
-                    if (searchBox.text === "") {
-                        toggleCategory(previousCategory)
-                        return
-                    }
-                    if (popup.currentCategory !== StatusGifPopup.Category.Search) {
-                        popup.previousCategory = popup.currentCategory
-                        popup.currentCategory = StatusGifPopup.Category.Search
+
+        ColumnLayout {
+            anchors.fill: parent
+            spacing: 0
+
+            Item {
+                property int headerMargin: 8
+
+                id: gifHeader
+                Layout.fillWidth: true
+                height: searchBox.height + gifHeader.headerMargin
```

Comment:
This won't work, you are inside a Layout

Reply:
not sure what won't work here. Everything seems to work as expected. ?

Reply:
Well specifying `height` inside a Layout doesn't work (if it does that's just by chance). You should use `Layout.preferredHeight` instead

Reply:
Good point, thanks!